bika/lims/browser/instrument.py

```diff
@@ -99,20 +112,22 @@ def folderitems(self):
         toshow = []
         for man in self.context.getMaintenanceTasks():
             toshow.append(man.UID())
-        for x in range (len(items)):
-            if not items[x].has_key('obj'): continue
-            obj = items[x]['obj']
+
+        for item in items:
```

Comment:
I see you've done here small refactoring, not related with the functionality itself, but would be much better if you use `folderitem` function instead of `folderitems` (with "s" at the end), cause this would prevent this extra loop.